docs/SetupTestEnvironment.md

```diff
@@ -0,0 +1,38 @@
+# Setup Test Environment
+
+## Create a Service Principal
```

Comment:
Why create a service principal here? AKS will create one for us. If this is for use with ACR, this should be after "Create an Azure Container Registry".